api-server/dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM python:3.9
 
+RUN apt-get update && apt-get install -y \
+    ghostscript
```

Comment:
is this a dependency for wand?

Reply:
yes, without this it doesn't write the DDS I have found.